server/src/main/java/org/opensearch/cluster/decommission/DecommissionStatus.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster.decommission;
+
+public enum DecommissionStatus {
+    /**
+     * Decommission process is initiated
+     */
+    INIT((byte) 0),
```

Comment:
should there be a state that signifies that the voting config exclusion was done or not?

Reply:
Didn't see any use case for that. Decommission process itself starts after abdication and checks if the operation is executed on a non decommissioned active master node